db/migrate/20190313102844_create_migrations.rb

```diff
@@ -27,5 +27,27 @@ def change
 
       t.timestamps
     end
+
+    create_table :users do |t|
```

Comment:
this should be in a different migration from the other one. The other one my have already been used in other environments if you worked with more people.